ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/evpn_role_server_overlay_vxlan_ibgp.yml

```
@@ -0,0 +1 @@
+fabric_evpn_encapsulation: vxlan
```

please add a comment
```suggestion
---
fabric_evpn_encapsulation: vxlan
```